manifest.yml

```diff
@@ -0,0 +1,8 @@
+---
+path: ./docs
+applications:
+- name: fec-pattern-library
+  memory: 64M
+  buildpack: https://github.com/cloudfoundry/staticfile-buildpack.git
```

Comment:
Any reason to download the buildpack from github instead of using the cached `staticfile_buildpack`?